File: Core python/Assignment/Assignment 3/q3.py
# Write a program to input angles of a triangle and check whether triangle is valid or not.

# take of input of angle of traingle
a = int(input("enter 1st angle : "))
b = int(input("enter 2st angle : "))
c = int(input("enter 3st angle : "))

# calculate angle sum
sum = a+b+c

# print output base on condition
if(sum == 180):
    print(f'is valid traingle')
else:
    print(f'is not valid traingle')

File: C/Assignments/Assignment 5/q3.c
/* 3. Print an inverted right-angled triangle pattern
Input: n = 5
Output:

*****
****
***
**
*

*/

void main() {
	
	int n ;
	printf("enter n : ");
	scanf("%d", &n);
	
	for(int i = 1 ; i <= n ; i++){
		for(int j=1 ; j <= n - i+1 ; j++){
			printf("* ");
		}
		printf("\n");
	}
}